csharp/src/ProtocolBuffers/Descriptors/FileDescriptor.cs

```diff
@@ -54,6 +54,17 @@ public sealed class FileDescriptor : IDescriptor<FileDescriptorProto>
         private readonly IList<FileDescriptor> publicDependencies;
         private readonly DescriptorPool pool;
 
+        public enum ProtoSyntax

```

Comment:
Sorry if my comment wasn't clear before - these enum values should be Proto2 and Proto3 (i.e. not shouty).